src/NuGet.Clients/NuGet.CommandLine/MsBuildUtility.cs

```diff
@@ -226,7 +227,8 @@ public static bool IsMsBuildBasedProject(string projectFullPath)
             string restoreConfigFile,
             string[] sources,
             string packagesDirectory,
-            MsBuildToolset toolset)
+            MsBuildToolset toolset,
+            RestoreLockProperties restoreLockProperties)
```

Comment:
nit: 
In methods with lots of params, I like `)` on a new line as it creates a cleaner diff as new params get added :)

Reply:
I typically ding your PR's for this.  :)  It's a trivial difference in diff.  If optimizing for cleaner diffs in trivial cases could lead to lots of formatting changes.

Reply:
I only do that if the argument has so many parameters where it warrants that they are each on a new line. 

Regardless don't want us to dwell on this :)